internal/store/driver.go

```diff
@@ -568,7 +701,74 @@ func (d *Driver) calculateHTTPSEdges() map[string]ingressv1alpha1.HTTPSEdge {
 		edge.Spec.Metadata = d.customMetadata
 		edgeMap[domain.Spec.Domain] = edge
 	}
+	d.calculateHTTPSEdgesFromIngress(edgeMap)
+
+	if d.gatewayEnabled {
+
+		httproutes := d.store.ListHTTPRoutes()
+		gateways := d.store.ListGateways()
+		for _, gtw := range gateways {
+			var gatewayDomains []string
+			for _, listener := range gtw.Spec.Listeners {
+				if listener.Hostname == nil {
```

Comment:
should we also skip the `ingress` domains?

Reply:
fixed here: https://github.com/ngrok/kubernetes-ingress-controller/pull/343/commits/be2602657a8d34fb40c46cdb36eb361884893ae9